man/sd_bus_call.xml

```diff
@@ -104,7 +104,12 @@
     <refsect2 id='errors'>
       <title>Errors</title>
 
-      <para>Returned errors may indicate the following problems:</para>
+      <para>Returned errors may indicate the following problems. When
```

Comment:
This sentence needs to be moved to the end.

Reply:
Thanks.